Updating Service Catalog PagerDuty integrations doc (#20589)
DataDog/documentation: content/en/tracing/service_catalog/integrations.md

## PagerDuty integration

> You can add PagerDuty metadata to a service so that the Service Catalog displays and links to information such as who is on-call and whether there are active PagerDuty incidents for the service. Only one on-call can be shown at a time, so the PagerDuty integration will pick the first user returned when sorting by escalation level, then email alphabetically.  

Reviewer: ```suggestion
You can add PagerDuty metadata to a service so that the Service Catalog displays and links to information such as who is on-call and whether there are active PagerDuty incidents for the service. Because only one on-call can be displayed, PagerDuty selects the first user by escalation level, then alphabetically by email.
```
Making Vale happy by trimming to <25 words and removing "will".

Reviewer 2: re: "PagerDuty selects the first user by escalation level, then alphabetically by email." -> it's actually "Datadog selects"

Reviewer: Thanks! Updated the suggestion.
